source/docs/casper/developers/prerequisites.md

```diff
@@ -1,4 +1,6 @@
 import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Development Prerequisites
 
```

Comment:
I would link these headings to the relevant sections

Reply:
What do you have in mind saying "link to relevant sections"? those are just React component imports so the Tabs can be used, the imports are always on top. Or did I misunderstand?

Reply:
Headings were linked to the appropriate sections